deploy/packages/parallel-libraries.yaml

```
@@ -85,7 +85,8 @@ packages:
       - mpi
       - python
     specs:
+      - neuron
       - neuron@7.6.8
-      - neuron+debug@7.6.8
+      - neuron+debug
```

I think that the version of `neuron` should be defined to avoid keeping multiple neuron versions built. 
Also the `neuron` without debug is defined two times now. Unless this is done for another reason.
```suggestion
      - neuron@7.8.0a
      - neuron+debug@7.8.0a
```

---

This was at some point discussed with Matthias. We should avoid setting versions here because this is actually a source of multiple versions getting built (because we ask to install a specific version, but dependencies dont see this and will typically request the latest). This being said, we should define the preferred version in the recipe directly, or, in cases it's not suitable, in the packages.yaml.
In the case of neuron the rationale is to keep the previous stable version available (which is anyway in the existing built tree) and the latest "preferred", and present two modules to the user.
For neuron+debug, that is mostly used for SimulationStack testing, and there it is useful to use the latest preferred release.

---

Agree...any reason to keep 7.6.8 version then?